build-tests/x86/rawhide/test-image-systemd-boot/config.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#======================================
+# Helpers
+#--------------------------------------
+# shellcheck disable=SC1091
+test -f /.kconfig && . /.kconfig
+
+declare kiwi_iname
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Fix kernel-install
+#--------------------------------------
+patch -p0 < kernel-install.patch
+rm -f kernel-install.patch
+
+#======================================
+# Activate services
+#--------------------------------------
+baseInsertService dbus-broker
+baseInsertService NetworkManager
+
+#======================================
+# Setup default target, multi-user
+#--------------------------------------
+baseSetRunlevel 3
```

Comment:
we really should have a `baseSetTarget` function instead, because we can't guarantee that the runlevel symlinks for targets exist.

Reply:
We did not change the name of the method for compat reasons. However, I think the code behind it does things correctly for systemd. Please double check: https://github.com/OSInside/kiwi/blob/master/kiwi/config/functions.sh#L530 if you disagree.

Regarding the naming I have no objections to add a baseSetTarget, the times of sysV init are mostly over and it could be added in addition to avoid the runlevel terminology. If this should be needed it can be done in a followup request but not here please

Reply:
Sure, I'm just noting we should probably have an alias or something here added at some point.

Reply:
yes agreed, I put it on my list :+1:

Reply:
https://github.com/OSInside/kiwi/issues/2206